sdk/test/NetStandard/IntegrationTests/IntegrationTests/General.cs

```diff
@@ -177,7 +183,7 @@ private static void TestServiceCallForManualClockCorrection(ClockSkewTestContext
         {
             var oldManualClockCorrection = AWSConfigs.ManualClockCorrection;
             var oldCorrectClockSkew = AWSConfigs.CorrectForClockSkew;
-            var oldClockSkewCorrection = context.Config.ClockOffset;
```

Comment:
I'd not change these tests yet, let's address them later when we investigate the issue with `ClockOffset`.

Reply:
I don't think there is an issue with `ClockOffset`, but we can be defensive here and change it back